docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -78,10 +78,12 @@ Roles may also include modules and other plugin types in a directory called ``li
 Storing and finding roles
 =========================
 
-By default, Ansible looks for roles in two locations:
+By default, Ansible looks for roles in the following locations:
 
+- in collections, if you are using them
 - in a directory called ``roles/``, relative to the playbook file
-- in ``/etc/ansible/roles``
+- in the configured roles path. The default search path is ``~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles``.
```

Comment:
Is the configured roles path the same thing linked to right below this? If so, perhaps clarify that by making it a :ref: here as well?